registry/handlers/tags.go

```diff
@@ -49,6 +51,51 @@ func (th *tagsHandler) GetTags(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	// do pagination if requested
+	q := r.URL.Query()
+	// get entries after latest, if any specified
+	if lastEntry := q.Get("last"); lastEntry != "" {
+		lastEntryIndex := sort.SearchStrings(tags, lastEntry)
+
+		// as`sort.SearchStrings` can return len(tags), if the
+		// specified `lastEntry` is not found, we need to
+		// ensure it does not panic when slicing.
+		if lastEntryIndex == len(tags) {
+			tags = []string{}
+		} else {
```

Comment:
if the last doesn't exist, should not be a bad request?

Reply:
We can do partial searches, using the lastEntryIndex, so it doesn't need to be a exact match, and therefore is not really a error as far as I understand :)